src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/Content/Controls/FlyoutSamplePage.xaml.cs

```diff
@@ -23,6 +23,25 @@ public FlyoutSamplePage()
 		{
 			this.InitializeComponent();
 		}
+
+		private void CloseFlyout(object sender, string buttonName)
+		{
+			var appBarButton = sender as AppBarButton;
+			// FindName not working on other platforms at the moment
+			// https://github.com/unoplatform/uno/issues/7258
+			var button = (Button)appBarButton.FindName(buttonName);
```

Comment:
FindName doesnt work is most likely caused by everything being inside a DataTemplate, rather this linked issue.
You can probably abuse the tag property to get the flyout button: 
```xml
Tag="{Binding ElementName=ShowFullScreenFlyoutButton}"
```

Reply:
Thank you a lot @Xiaoy312 for the Tag proposition, I updated the code.
FindName doesn't work only on the other platforms, on UWP it's correctly finding the button. Something that we should align with at some point by fixing FindName in Uno.